src/Workspaces/CSharp/Portable/Formatting/Rules/SpacingFormattingRule.cs

```diff
@@ -205,15 +197,21 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
             }
 
             // For spacing delimiters - after comma
-            if ((previousToken.IsCommaInArgumentOrParameterList() && currentKind != SyntaxKind.OmittedTypeArgumentToken) ||
-                previousToken.IsCommaInInitializerExpression())
+            if ((previousToken.IsCommaInArgumentOrParameterList() && currentKind != SyntaxKind.OmittedTypeArgumentToken)
+                || previousToken.IsCommaInInitializerExpression()
+                || (previousKind == SyntaxKind.CommaToken
```

Comment:
Can you extract this condition into a method? Readability decreases with more conditions within `if`